modules/ob-verifying-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,17 @@
+// This module is included in the following assembly:
+//
+// * configuring/using-builds-in-a-restricted-environment.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-verifying-cluster-wide-proxy_{context}"]
+= Verifying cluster-wide proxy
+
+Run the following command to verify if the cluster-wide proxy is correctly configured:
```

Comment:
```suggestion
.Procedure
* To verify if the cluster-wide proxy is correctly configured, run the following command:
```

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command